desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -188,6 +188,10 @@
     public static final String DICTIONARY_TEXTFILE_IMAGEPATH = "images/model/dictionary_textfile.png";
     public static final String DICTIONARY_DATASTORE_IMAGEPATH = "images/model/dictionary_datastore.png";
 
+    public static final String NEWS_CHANNEL = "images/news/news_channel_16.png";
+    public static final String NEWS_CHANNEL_ORANGE = "images/news/news_channel_28.png";
+    public static final String NEWS_CHANNEL_HIGHLIGHTED = "images/news/news_channel_20.png";
```

Comment:
I agree with the bad filenames and I will put the images to graphic project. But a graphic expert explained to me that a resizable icons reduce readability icons by blurring. In the end she have created the icons in a selected size and in better quality than I can do. :-)